### PR TITLE
first challenge DOMAPI

### PR DESCRIPTION
**Pull request by nilete22**

### CODE REVIEW

File: challenge.md [hunk 1]
```diff
@@ -3,13 +3,21 @@
 1. Answer the following questions:
 
    - How would you select from JavaScript an element `p` that has the class `text` and also the class `important`?
+    document.querySelector('p.text.important');
```
Comment (tictools) [inline challenge.md:6]: 👌

File: challenge.md [hunk 2]
```diff
@@ -3,13 +3,21 @@
...
+
    - How would you select from JavaScript a `button` element with class `button` and that is disabled?
+    document.querySelector('button.button:disabled');
```
Comment (tictools) [inline challenge.md:9]: 👌

This selector also accepts this syntax:
```
document.querySelector(".button:disabled");
```

File: challenge.md [hunk 3]
```diff
@@ -3,13 +3,21 @@
...
+
    - How would you select from JavaScript all the `li` elements that are direct children of an `ul` element with class `list`?
+    document.querySelectorAll('ul.list > li');
```
Comment (tictools) [inline challenge.md:12]: 👌

File: challenge.md [hunk 4]
```diff
@@ -19,6 +27,11 @@
       FullStack Web Development.
     </p>
   </section>
+  <script>
+    const subtitle = document.querySelector('h2');
```
Comment (tictools) [inline challenge.md:31]: If possible, selecting elements using className is always the best option.

```suggestion
    const subtitle = document.querySelector('.subtitle');
```

File: challenge.md [hunk 5]
```diff
@@ -19,6 +27,11 @@
...
+    subtitle.textContent = 'The MERN stack';
+    subtitle.classList.remove('subtitle');
+  </script>
```
Comment (tictools) [inline challenge.md:34]: 👌

File: challenge.md [hunk 6]
```diff
@@ -30,6 +43,22 @@
   <span class="student-age"></span> years
   <img class="student-photo" src="" alt="" />
 </article>
+<script>
+    const student = {
+      name: "Nil Farras",
+      age: 18,
+      photoUrl: "https://t3.ftcdn.net/jpg/05/06/55/22/360_F_506552238_j3Y4oq4rrlLEgzVG30AdEe0TaRINtUKr.jpg"
+    };
```
Comment (tictools) [inline challenge.md:51]: 🔝

File: challenge.md [hunk 7]
```diff
@@ -30,6 +43,22 @@
...
+
+    const name = document.querySelector('.student-name');
+    const age = document.querySelector('.student-age');
+    const photo = document.querySelector('.student-photo');
+  
+    name.textContent = student.name;
+    age.textContent = student.age;
+    photo.src = student.photoUrl;
+    photo.alt = student.name + "'s photo";
+  </script>
```
Comment (tictools) [inline challenge.md:61]: 👌

File: challenge.md
Comment (tictools) [inline challenge.md:1]: Fix `subtitle` selector and push changes.
Then this PR can be merged.

File: challenge.md (hunk 4 above)
Comment (nilete22) [reply challenge.md:31]: Yes but if we need to delete it, i thought to don't selected by the class

Comment (tictools) [reply challenge.md:31]: @nilete22 
Sure, but removing class is next step. You can use class to select the element to ensure you are selecting the correct one. Other `h2` could exist in the nodes tree... then you probably will select the wrong one.

Using class name as selector is always safer than using the tag element itself.

**Pull request merged**